### DISCUSSION TITLE
Could you please add some binaries

### DISCUSSION THREAD

**Discussion by wishmaster86** [ipxe/ipxe#201] [Q&A]
Hi, As the Rom-o-Matic site is down for unknown how long. is it maybe possible to put up some completed binaries on your site?
Currently we can download from boot.ipxe.org but these are light versions. I need versions with all drivers in them to resolve a problem at our schools.

Kind regards,
Wishmaster86

**Comment by NiKiZe** [comment]
Both iPXE.efi and ipxe.pxe are full, which other do you need? (Double checking, all except for snponly.efi is full)

Still do setup your own build machine! You can even build it from wsl in windows 10.

**Reply by wishmaster86** [reply]
Hi and thanks for your reply. I am searching for the Undionly.kpxe with full drivers, http and menu support, thats all I need. 
it's not that I do not want to build it, I just dont have the time to set that up nore do I have the knowledge. 
Normally I use the Rom-O-Matic page to create it in a jiffy.

So therefore I was hoping somebody in the community or the creator himself would be so kind to share it.
Kind regards,
Wishmaster86

**Reply by NiKiZe** [reply]
Undiinly only has undi driver, (the name is.the driver)

Http and menu is also enabled by default.

Have you tried any of the existing ones?

**Reply by wishmaster86** [reply]
the one I have now is 66kb default undionly.kpxe from boot.ipxe.org and is not loading the menu, it goes directly to the harddisk. I have another one made by someone else 300+kb but its specific to his environment so I can not use it because he used an embedded script, But it did load his menu. He said that his version has all drivers in them made from the rom-o-matic site, So that is how i know I would need that default version with all drivers. Or can I edit this file somehow to load the DHCP boot file offered?

Kind regards,
Wishmaster86

**Reply by wishmaster86** [reply]
also I know that the undionly.kpxe works for other machines. It's just 1 type HP Probook 655 G2 that has Realtek card in it that is not supported in the current undionly.kpxe file that we are using.

**Reply by NiKiZe** [reply]
Again you can not have undionly with all drivers.

Please try ipxe.pxe
Undi relies on the NIC ROM, for integrated NICs that is your BIOS, so try to upgrade that to get a pure undionly build working.

**Reply by sm1000** [reply]
Try to build your own version [here](https://rom-o-matic.dev/).
